EDDI/EDDI.cs

```diff
@@ -1678,12 +1685,14 @@ private bool eventShipyard(ShipyardEvent theEvent)
                             ships.Add(ship);
```

Comment:
This block could be simplified as above.

Reply:
ok